### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by nu11secur1ty**
Adding a red icons, to the dark theme.

### CODE REVIEW

**Comment by nu11secur1ty** [comment]
Thanks

File: themes/index.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 export const themes = {
+  dark_red: {
```
Comment (rickstaa) [inline themes/index.js:2]: Thanks for your contribution. I'm happy to merge this theme if you put the code under the `dark` theme 👍🏻.

**Review by rickstaa — CHANGES_REQUESTED** [review]

File: themes/index.js (hunk 1 above)
Comment (nu11secur1ty) [reply themes/index.js:2]: Thank you <3

**Comment by anuraghazra** [comment]
We are currently pausing addition of new themes. If this theme is exclusively for your personal use, then instead of adding it to our theme collection, you can use card [customization options](https://github.com/anuraghazra/github-readme-stats#customization).

**Pull request closed without merging**